### PR TITLE
win32: Add -in-tree option in edje_cc for Windows

### PR DESCRIPTION
**Pull request by joaoantoniocardoso**
This PR fixes the following error when building elementary:
```
[2871/3123] Generating edje_cc_lua_script.edc with a meson_exe.py custom command
FAILED: src/examples/edje/lua_script.edj
edje_cc.exe: Error. Cannot run epp: c:/lib/edje/utils/v-1.24/epp.exe
```

### CODE REVIEW

File: src/tests/edje/tests/meson.build [hunk 1]
```diff
@@ -6,8 +6,8 @@ foreach edc_file : edc_files
    themes += custom_target('edje_cc_' + edc_file,
     input : edc_file,
     output : '@BASENAME@.edj',
-    command : edje_cc_exe + [ '-beta', '-fastcomp',
-              '-id', join_paths(meson.source_root(), 'src', 'tests', 'emotion', 'data'),
+    command : edje_cc_exe + [ '-beta', '-fastcomp', '-in-tree',
+              '-id', join_paths(meson.source_root(), 'src', 'tests', 'data'),
```
Comment (JPTIZ) [inline src/tests/edje/tests/meson.build:10]: ?

File: src/bin/edje/edje_cc.c [hunk 2]
```diff
@@ -195,6 +197,11 @@ main(int argc, char **argv)
           {
              eina_log_domain_level_set("edje_cc", EINA_LOG_LEVEL_INFO);
           }
+        else if (!strcmp(argv[i], "-in-tree"))
+          {
+             putenv("EFL_RUN_IN_TREE=1");
+             in_tree = 1;
```
Comment (JPTIZ) [inline src/bin/edje/edje_cc.c:203]: Maybe we could try a slightly different approach: we rely on `in_tree` global variable (which could be renamed to `efl_run_in_tree`) and, in the start of `main`, check for `getenv(EFL_RUN_IN_TREE)` is not 0 and if that's the case then set `efl_run_in_tree` to 1.

File: src/tests/edje/tests/meson.build (hunk 1 above)
Comment (JPTIZ) [reply src/tests/edje/tests/meson.build:10]: Ok, just a plain "?" isn't a proper way to ask things, so...

Why exactly are we removing `emotion` there?

File: src/bin/edje/edje_cc.c (hunk 2 above)
Comment (Coquinho) [reply src/bin/edje/edje_cc.c:203]: Sounds better for me too.

File: src/tests/edje/tests/meson.build (hunk 1 above)
Comment (joaoantoniocardoso) [reply src/tests/edje/tests/meson.build:10]: Hmm yeah, that seems wrong to me. I will take a look at all the other places too

File: src/bin/edje/edje_cc.c (hunk 2 above)
Comment (joaoantoniocardoso) [reply src/bin/edje/edje_cc.c:203]: Nice, I'll try that.

File: src/tests/edje/tests/meson.build (hunk 1 above)
Comment (Coquinho) [reply src/tests/edje/tests/meson.build:10]: Maybe this should be at #369 ?

File: src/tests/edje/data/meson.build [hunk 3]
```diff
@@ -25,7 +25,8 @@ foreach edc_file : edc_files
    themes += custom_target('edje_cc_' + edc_file,
     input : edc_file,
     output : '@BASENAME@.edj',
-    command : edje_cc_exe + [ '-beta', '-fastcomp',
+    command : edje_cc_exe + [ '-beta', '-fastcomp', '-in-tree',
+              '-id', join_paths(meson.source_root(), 'src', 'tests', 'data'),
```
Comment (joaoantoniocardoso) [inline src/tests/edje/data/meson.build:29]: @JPTIZ here it is another odd change

Comment (Coquinho) [reply src/tests/edje/data/meson.build:29]: this probably should be to remove emotion too

File: src/tests/edje/tests/meson.build (hunk 1 above)
Comment (joaoantoniocardoso) [reply src/tests/edje/tests/meson.build:10]: actually there is no `src/tests/data` in tree.

File: src/tests/edje/data/meson.build (hunk 3 above)
Comment (joaoantoniocardoso) [reply src/tests/edje/data/meson.build]: > actually there is no src/tests/data in tree, I'll keep only src/tests/emotion/data here and everywhere else.

**Pull request merged**